.github/workflows/markdown.links.config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "replacementPatterns": [
+      {
+        "pattern": "^/",
+        "replacement": "https://keptn-experimental-docs-site.netlify.app/"
+      }
+    ],
+    "ignorePatterns": [
+      {
+        "pattern": "^https?://localhost"
```

Comment:
Why do we want to ignore this?

Reply:
@vadasambar I am ignoring this because I think there might be a case when we use the `localhost` keyword in the documentation.